gapic/templates/tests/unit/gapic/%name_%version/%sub/test_macros.j2

```diff
@@ -1947,3 +1929,91 @@ def test_unsupported_parameter_rest_asyncio():
     )
 
 {% endmacro %}
+
+{# is_rest_unsupported_method renders:
+ # 'True' if transport is async REST.
+ # 'True' if transport is sync REST and method is a client streaming method.
+ # 'False' otherwise.
+#}
+{# NOTE: We will keep updating this method as we add support for methods in async REST. #}
+{% macro is_rest_unsupported_method(method, is_async) %}
```

Comment:
You're right. `is_supported` would've been a better choice and less confusing. I'll be keeping it how it is now since I expect it to be removed once the async rest work is completed.